### ISSUE TITLE
Messages containing HTML entities such as &nbsp; cause a MessageParserException

### ISSUE THREAD

**Issue by jimbodunc** [finos/symphony-bdk-python#207]
Parsing any message containing HTML entities fails in the function `get_text_content_from_message`:

https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/blob/a15462a88ea21bc95c5ca28d92f5069c1a0d7369/symphony/bdk/core/service/message/message_parser.py#L24-L28

This issue is that **difusedxml** won't accept non-XML entities by default.

This is a particular issue because tables copy/pasted from Excel into a symphony chat render empty cells as `&nbsp;`

For example:
![image](https://user-images.githubusercontent.com/87011074/124580371-f3d81100-de47-11eb-83bd-93d90d427a73.png)

Arrives as:

```xml
<div data-format="PresentationML" data-version="2.0" class="wysiwyg">
    <p>
        <table class="pasted-table">
            <thead>
                <tr>
                    <th>/bilat-barc-dev</th>
                    <th>Bid</th>
                    <th>Mid</th>
                    <th>Ask</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>&nbsp;</td>
                    <td>1.114</td>
                    <td>1.893</td>
                    <td>0.291</td>
                </tr>
            </tbody>
        </table>
    </p>
</div>
```

**Comment by jimbodunc** [comment]
I noticed that the same table pasted into Symphony 1.5 does not contain `&nbsp;` so this might (also) be a platform bug.

```xml
<div data-format="PresentationML" data-version="2.0" class="wysiwyg">
    <p>
        <table class="pasted-table">
            <thead>
                <tr>
                    <th>/bilat-barc-dev</th>
                    <th>Bid</th>
                    <th>Mid</th>
                    <th>Ask</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>&#160;</td>
                    <td>1.114</td>
                    <td>1.893</td>
                    <td>0.291</td>
                </tr>
                <tr>
                    <td>2</td>
                    <td>1.265</td>
                    <td>1.304</td>
                    <td>1.727</td>
                </tr>
            </tbody>
        </table>
    </p>
</div>
```

**Comment by symphony-mariacristina** [comment]
Hello @jimbodunc! thanks for highlighting this issue, your PR has been merged.
Thanks for your contribution!

**Comment by symphony-mariacristina** [comment]
@jimbodunc we released a new BDK 2.0 version, your fix is now available on 2.0b5 version https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/releases/tag/2.0b5

**Comment by jimbodunc** [comment]
@symphony-mariacristina Great, thanks!
